CodeStarterCamp_Week1/main.swift

```diff
@@ -8,20 +8,23 @@
 
 import Foundation
 
-func drawice(ice_height:Int){
-    for i in 1...ice_height{
+func drawIce(height:Int){
+    for _ in 1...height{
         for _ in 1...11{
             print("*",terminator: "")
         }
         print()
     }
 }
```

Comment:
띄어쓰기에 조금 신경써보시면 어떨까 싶네요!
변수 다음에 { 가 올때, 그리고 메서드의 파라미터 콜론(:) 뒤, 메서드 파라미터를 구분하는 콤마(,) 뒤 → 이런 곳들에서요 ㅎㅎㅎ

이런 부분들이 귀찮다면, 컨벤션을 자동으로 맞춰주는 툴도 있답니다! [SwiftLint](https://github.com/realm/SwiftLint) 라는 건데, 나중에 관심있으시면 한 번 살펴보세요 👍🏻 

```suggestion
func drawIce(height: Int) {
    for _ in 1...height {
        for _ in 1...11 {
            print("*", terminator: "")
        }
        print()
    }
}
```